src/languages/en.js

```diff
@@ -320,6 +320,7 @@ export default {
         settlePaypalMe: 'Pay with PayPal.me',
         requestAmount: ({amount}) => `request ${amount}`,
         splitAmount: ({amount}) => `split ${amount}`,
+        payerOwe: ({payer, amount}) => `${payer} owes ${amount}`,
```

Comment:
this is such a minor point. But could this be `payerOwesAmount`? That's the format I'm using in my related PR